src/components/DatePicker/DatePicker.tsx

```diff
@@ -1,27 +1,27 @@
 import React, {useState, useEffect, useCallback, useMemo} from 'react';
 import {ArrowLeftMinor, ArrowRightMinor} from '@shopify/polaris-icons';
+
+import {Button} from '../Button';
+import {classNames} from '../../utilities/css';
 import {
-  Range,
-  Months,
-  Year,
+  Month as Months,
```

Comment:
This dance with the renaming and export seems a bit confusing

Reply:
Yeah it's horrible :( In this initial commit i was trying to maintain backwards compatibility and satisfy 3 things:
- linting says to prefer singular enums (so Month, not Months), so name it right in the first place
- We have a public export called Months, and that API compatibility should be kept (for the moment, with renaming that in either a follow up either in this PR or in a new one)
- DatePicker already has an import called Month

Looking at this has made me realize that our DatePicker API isn't that nice and it's i18n support leaves something to be desired.

I chatted with @kaelig and we were thinking:

- Replace typings in DatePicker that reference `Month`, `Year` and `Weekday` with plain old `number`. As you mention below Year isn't a useful abstraction. And the `Month` / `Weekday` enums don't add a great deal either:  `Month.February` instead of `1` and `Weekday.Monday` instead of `1` as alternatives are slightly nicer but not nice enough to warrant having to have an extra export.
- the Month and Weekdays enums don't a lot of value - remove them